frontend/package.json

```diff
@@ -30,6 +30,7 @@
   "devDependencies": {
     "@rushstack/eslint-patch": "^1.2.0",
     "@types/node": "^16.18.10",
+    "@vitejs/plugin-basic-ssl": "^1.0.1",
```

Comment:
Wanted to keep this one in. I wasn't able to successfully test Metamask mobile because of the issue you mentioned, but was trying to recompile the app to support it. Will remove.